Dockerfile

```diff
@@ -1,9 +1,10 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.11 AS builder
-WORKDIR /go/src/github.com/nutanix-cloud-native/machine-api-provider-nutanix
+
+FROM registry.ci.openshift.org/openshift/release:golang-1.17 AS builder
+WORKDIR /go/src/github.com/openshift/machine-api-provider-nutanix
 COPY . .
 # VERSION env gets set in the openshift/release image and refers to the golang version, which interfers with our own
-#RUN make build
+#RUN NO_DOCKER=1 make build
 RUN unset VERSION && GOPROXY=off NO_DOCKER=1 make build
```

Comment:
Why do we unset the version?

Reply:
I see all other providers (aws, gcp, azure, ibmcloud, etc) all use this in their Dockerfile.

Reply:
Fair, I wonder why 🤔  Not a blocker